views/layout.jade

```diff
@@ -9,6 +9,15 @@ html
   body.bg-dark
     nav.clearfix.bg-light.white.sm-show
       div.container
+        //- if messages.error
```

Comment:
Cool cool -- I dig it. Given that express-flash provides the `messages` object on `locals`, I think we can safely uncomment these lines, right? I think it makes sense to keep them in -- we'll definitely need them :)